tests/integration/cgroups.bats

```diff
@@ -6,24 +6,24 @@ function teardown() {
     rm -f $BATS_TMPDIR/runc-cgroups-integration-test.json
     teardown_running_container test_cgroups_kmem
     teardown_running_container test_cgroups_permissions
-    teardown_busybox
+    teardown_debian
```

Comment:
Can we rename functions like these to be distro agnostic so we can change the implementation to use a different distro more isolated in the future?

Reply:
You can also rename `$DEBIAN_BUNDLE` to `$BUNDLE` for the same reason.

Reply:
I can do that!

Reply:
Taken care of. seems to be running locally, working through the CI now!